serverless.yml

```diff
@@ -1,82 +1,100 @@
 service: dlt-functions
 provider:
-    name: aws
-    runtime: nodejs8.10
-    stage: '${opt:stage, ''stg''}'
-    region: ap-southeast-1
-    environment:
-        BUCKET_NAME: '${self:custom.BUCKET_NAME.${self:provider.stage}}'
-        SES_KEY_ID: '${self:custom.SES_KEY_ID.${self:provider.stage}}'
-        SES_SECRET: '${self:custom.SES_SECRET.${self:provider.stage}}'
-        SES_REGION: '${self:custom.SES_REGION.${self:provider.stage}}'
-        RECAPTCHA_SECRET: '${self:custom.RECAPTCHA_SECRET.${self:provider.stage}}'
-        NETWORK: '${self:custom.NETWORK.${self:provider.stage}}'
-        EMAIL_API_KEYS: '${self:custom.EMAIL_API_KEYS.${self:provider.stage}}'
-    iamRoleStatements:
-        -
-            Effect: Allow
-            Action: ['s3:putObject', 's3:upload', 's3:deleteObject', 's3:getObject']
-            Resource: { "Fn::Join" : ["", ["arn:aws:s3:::", { "Ref" : '${self:custom.BUCKET_NAME.${self:provider.stage}}' } ] ]  }
+  name: aws
+  runtime: nodejs8.10
+  stage: '${opt:stage, ''stg''}'
+  region: ap-southeast-1
+  environment:
+    BUCKET_NAME: '${self:custom.BUCKET_NAME.${self:provider.stage}}'
+    SES_KEY_ID: '${self:custom.SES_KEY_ID.${self:provider.stage}}'
+    SES_SECRET: '${self:custom.SES_SECRET.${self:provider.stage}}'
+    SES_REGION: '${self:custom.SES_REGION.${self:provider.stage}}'
+    RECAPTCHA_SECRET: '${self:custom.RECAPTCHA_SECRET.${self:provider.stage}}'
+    NETWORK: '${self:custom.NETWORK.${self:provider.stage}}'
+    EMAIL_API_KEYS: '${self:custom.EMAIL_API_KEYS.${self:provider.stage}}'
+  iamRoleStatements:
+    -
+      Effect: Allow
+      Action: ['s3:putObject', 's3:upload', 's3:deleteObject', 's3:getObject']
+      Resource: { "Fn::Join" : ["", ["arn:aws:s3:::", { "Ref" : '${self:custom.BUCKET_NAME.${self:provider.stage}}' } ] ]  }
 resources:
   Resources:
-    DevBucket:
-      Type: AWS::S3::Bucket
-      Properties:
-        BucketName: 'dev-bucket'
     DeployedBucket:
       Type: AWS::S3::Bucket
       Properties:
         BucketName: '${self:custom.BUCKET_NAME.${self:provider.stage}}'
+        AccessControl: PublicRead
```

Comment:
as discussed, only the function needs access to the bucket

Reply:
removed